doc/data/messages/t/too-many-public-methods/bad.py

```diff
@@ -0,0 +1,21 @@
+class Foo: # [too-many-public-methods]
```

Comment:
Can you use something else than foo or bar plz ? Use a concept a 8 year old would understand (because there will be 8 year old reading this).

Reply:
Isn't this supposed to be about moving examples from pylint-errors to pylint?

Reply:
This is also about making the example the best we can in the process. (There's no way to know if an example come from pylint-errors, you should credit the original implementer in your own commit if it does.)

Reply:
Also:

1. I think the code would be easier to understand if the error message was not at the top of the code and was at the line where the the code crosses the max-public-methods number.
2. I also think that if an 8 year old were programming then they would be getting help from an adult or a programming book.